### PR TITLE
Added sorting mechanism for recommendations

### LINKED ISSUE #161: Sorting Assets by Maintenance recommendation
As a user, I want to be able to sort the assets by maintenance recommendation.

Acceptance Criteria:
- [ ] Update documentation for mock-ups on Confluence (link to be provided in the issue)
- [ ] Sorting button/option on the systems thumbnails view
- [ ] The assets in the thumbnail view can be sorted by maintenance recommendation (ascending or descending criticality)
- [ ] Unit tests

### PR DESCRIPTION
**Pull request by rdsaliba**

### CODE REVIEW

**Review by TalalBazerbachi — DISMISSED** [review]
Remove unused objects in AssetsController

File: Server/src/main/java/app/item/Asset.java [hunk 1]
```diff
@@ -115,6 +125,24 @@ public void setAssetTypeName(String assetTypeName) {
...
+                return 1;
+            case ADVISORY_THRESHOLD:
+                return 2;
+            case CAUTION_THRESHOLD:
+                return 3;
+            case WARNING_THRESHOLD:
+                return 4;
+            case FAILED_THRESHOLD:
+                return 5;
+
```
Comment (Shirwarsame) [inline Server/src/main/java/app/item/Asset.java:140]: Remove white space

**Review by Shirwarsame — COMMENTED** [review]
All values are -10000 (that might have been discussed before) making hard to tell if sorting is working.
Missing unit tests.

**Review by Shirwarsame — APPROVED** [review]
Issues discussed

**Review by TalalBazerbachi — APPROVED** [review]
UI is working as intended, and the assets are sorted in terms of recommendation

**Pull request merged**